guides/user/plugin-users/index.md

```diff
@@ -11,78 +11,191 @@ sidebar:
   <strong>Note:</strong> Plugins are an early alpha feature that is under active development and will likely change.
 </div>
 
-This guide is for adding existing plugins to Spinnaker. For information about how to develop a new plugin, see [Plugin Creators Guide](/guides/developer/plugin-creators). 
+In this guide, you add an existing plugin from an [example repository](https://github.com/spinnaker-plugin-examples/examplePluginRepository) to Spinnaker. See the [Plugin Creators Guide](/guides/developer/plugin-creators) for how to create a PF4J or Spring plugin.
+
+# Requirements
+
+* The plugin is either a PF4J or a Spring plugin
+* The plugin resides in a publicly accessible place
+* Spinnaker v1.19 or later
+* Halyard v1.32.0 or later
+* You redeploy Spinnaker using Halyard to apply changes
+
+# Define Plugins
+
+Define plugins in a file called `plugins.json`. This guide uses the  [file](https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json) in the example repository.
+
+```json
+[
+ {
+   "id": <plugin-id>,
+   "description": <description>,
+   "provider": <provider>,
+   "releases": [
+	 {
+	   "version": <version>,
+	   "date": <date>,
+	   "requires": <comma-delimited-list-of-spinnaker-services>,
+	   "sha512sum": <checksum>,
+	   "state": <state>,
+	   "url": <complete-url-to-bundle-zip-file>
+	 }
+   ]
+ }
+]
+```
+
+1. **id**: defined by the plugin creator; find this value in the bundle's `MANIFEST.MF`; unique value
+2. **description**: what the plugin does
+3. **provider**: plugin provider name
+4. **releases**: one-to-many list; each release entry requires:
```

Comment:
"one-to-many"
Do you mean "one or more"?

Reply:
yes. one-to-many is OO developer speak. sorry.

Reply:
removed list